test/semantics/call02.f90

```diff
@@ -63,22 +63,6 @@ elemental real function elem04(x)
 end module
 
 module m02
-  interface
-    subroutine altreturn(*)
-    end subroutine
-  end interface
- contains
-  subroutine test
-1   continue
-   contains
-    subroutine internal
-      !ERROR: alternate return label must be in the inclusive scope
-      call altreturn(*1)
```

Comment:
Is this check still a TODO? Should the test be moved to a different call??.f90?

Reply:
It was redundant; label validation catches the error, but then none of the other semantic checks are applied.